### DISCUSSION TITLE
Passages Contribution

### DISCUSSION THREAD

**Discussion by a-nu-ragn** [monkeytypegame/monkeytype#3467] [Questions] [answered]
Want to contribute to the phrases and passages we're typing on the website? How can I do that? 
And if possible please add a sample file image.

**Comment by mak448a** [comment]
Submit quote here
![monkeytype](https://user-images.githubusercontent.com/94062293/188973145-d8aa9956-39b4-4ddf-89c9-c185f65af63d.png)

**Comment by Miodec** [comment, marked as answer]
Submitting quotes through the website (like @mak448a suggested) has been temporarily disabled due to a large queue. You can still submit a pull request if you want to.

**Reply by a-nu-ragn** [reply]
Do I have to write my quote inside the code block? 
like this
{
      "text": "I'd tell you all you want and more if the sounds I made could be what you hear.",
      "source": "Infinite Jest",
      "id": 97,
      "length": 80
    },
